app/controllers/cloud_tenant_controller.rb

```diff
@@ -7,58 +7,24 @@ class CloudTenantController < ApplicationController
 
   include Mixins::GenericListMixin
   include Mixins::CheckedIdMixin
+  include Mixins::GenericButtonMixin
 
   # handle buttons pressed on the button bar
   def button
-    @edit = session[:edit]                                  # Restore @edit for adv search box
-    params[:display] = @display if %w(vms instances images).include?(@display)
```

Comment:
Potential difference .. `vms instances images` vs `images instances` in GenericButtonMixin..